doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,2 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
I would also add `etc/migrations/.gitkeep` to the PR so that the directory is automatically created when installing the bundle.